### DISCUSSION TITLE
ERR happen when use google analytics

### DISCUSSION THREAD

**Discussion by icebilly** [squidfunk/mkdocs-material#4328] [Q&A] [answered]
Hi，I set google analytics in yml like this:
extra:
  analytics:
    provider: google
    property: G-MYGOOGLETAGID

Then deploy the website on the server.
I will get error when visit the website, it seems that cannot reach to https://www.googletagmanager.com/gtag/js
![image](https://user-images.githubusercontent.com/21252620/188799484-937d190d-214a-4013-b4f9-280fe41c64fb.png)

So, my google tag cannot get any data from the website.
![image](https://user-images.githubusercontent.com/21252620/188808960-414eaf96-6d6e-42a1-853c-1d121487ebb0.png)

**Comment by squidfunk** [comment, marked as answer]
This seems to be a configuration error with Google Analytics. I suggest you post your problem on StackOverflow because it has nothing to do with this project. All we do is integrate Google Analytics, and from what you shared, that seems to be working.
